src/app/portlets/dot-edit-page/content/dot-edit-content.component.ts

```diff
@@ -150,6 +151,16 @@ export class DotEditContentComponent implements OnInit, OnDestroy {
             });
     }
 
+    private pageHasLayout() {
+        return this.pageState && this.pageState.layout;
+    }
+
+    private changeContainersHeight() {
+        if (this.pageHasLayout() && this.pageState.state.mode === 0) {
```

Comment:
The page state is a enum, use it to compare: ```=== PageMode.EDIT```

Also, ```pageHasLayout``` can be renamed like: ```shouldSetContainersHeight``` and group both conditions (unless we use ```pageHasLayout``` in another place)